force-app/tests/DataWeaveInApex Recipes/BasicDataWeaveInApexRecipes_Tests.cls

```diff
@@ -0,0 +1,39 @@
+@isTest
+public class BasicDataWeaveInApexRecipes_Tests {
+    @isTest
+    static void testStaticInvocation() {
+        Test.startTest();
```

Comment:
Do we need this for DataWave only execution?

Reply:
You mean `Test.startTest()` here?

Reply:
Its generally a best practice to have a  `Test.startTest()` and `Test.stopTest()` to reset the governor limit.